### DISCUSSION TITLE
Paying for my transaction is difficult I am new to BTC...

### DISCUSSION THREAD

**Discussion by AlmedaLM** [btcpayserver/btcpayserver#5162] [Technical Support]
I have been trying to figure out how to pay for my BTC transfer of $64 and I could sure use some help on this transaction...
Also why doesn't the BTC server just subtract the transaction fee with my BTC before transferring it to my wallet ? That makes it much easier for anyone...
Thank You Sincerely Almeda McKinney...

**Comment by dennisreimann** [comment]
Please don't pay anything, see details about the scam here https://blog.btcpayserver.org/bitcoin-bonus-scam/
